app/Controllers/feedController.php

```diff
@@ -647,6 +647,150 @@ public function deleteAction() {
 		}
 	}
 
+	/**
+	 * This action force clears the cache of a feed.
+	 *
+	 * Parameters are:
+	 *   - id (mandatory - no default): Feed ID
+	 *
+	 */
+	public function clearcacheAction() {
```

Comment:
```suggestion
	public function clearCacheAction() {
```
And corresponding calls